app/components/primer/alpha/banner.rb

```diff
@@ -104,7 +104,8 @@ def initialize(full: false, full_when_narrow: false, dismissible: false, descrip
           LEGACY_SCHEME_MAPPINGS[@scheme],
           "Banner--full": full,
           "flash-full": full, # legacy
-          "Banner--full-whenNarrow": full_when_narrow
+          "Banner--full-whenNarrow": full_when_narrow,
+          "Banner--centered": full && description && description.length > 120 # center with max-width for long descriptions
```

Comment:
I couldn't figure out how to do the same for `content`. Doing:

```
full && content && content.length > 120
```

results in the content not rendering? Is it because content can contain HTML? 🤔

Reply:
@simurai I think that's probably because `content` isn't available until render time. Calling it in the constructor is probably causing a `nil` or blank string to be cached for the content, meaning any subsequent calls to `#content` will return `nil` too. Can you add this class in `before_render` instead?

Reply:
> Can you add this class in `before_render` instead?

Yes, that did the trick. Thanks. 🙇